pkg/otbuiltin/commit.go

```diff
@@ -412,6 +415,29 @@ func parseKeyValueStrings(pairs []string) (*C.GVariant, error) {
 	builder := C.g_variant_builder_new(C._g_variant_type(C.CString("a{sv}")))
 	defer C.g_variant_builder_unref(builder)
 
+	// Add Ref bindings
+	sort.Strings(options.RefBindings)
```

Comment:
Eh this is a function to go from a generic keyvalue pair strings to a GVariant, It really shouldn't have this code added to it.. 

Was there a rebase accident here or ? Tbh i'm surprised you can access `options` here; but in any case if you need a less generic conversion function please make it that rather then doing a generic key value converstion and adding in a bunch of extra unexpected bits on the side :)

Reply:
Hope new iteration looks better.